common/ts/publish-task.ts

```diff
@@ -56,3 +34,27 @@ export default async function publishTask(endpointType: EndpointType) {
 function timeoutInSeconds(): number {
   return Number.parseInt(tl.getInput('pollingTimeoutSec', true), 10);
 }
+
+export async function getReportForTask(taskReport: TaskReport, metrics: Metrics, endpoint: Endpoint, timeoutSec: number): Promise<string> {
+  try {
+    const task = await Task.waitForTaskCompletion(endpoint, taskReport.ceTaskId, timeoutSec);
+    const analysis = await Analysis.getAnalysis({
+      analysisId: task.analysisId,
+      dashboardUrl: taskReport.dashboardUrl,
+      endpoint,
+      metrics,
+      projectName: task.componentName
```

Comment:
This is the only line of the product code that has changed. Previously there was a conditional check so the project name was only included if there were multiple projects.

Reply:
In the end I'm not sure you should change that... it will make the widget take more space for "nothing". The PR https://github.com/SonarSource/sonar-scanner-vsts/pull/38 was already merged to fix VSTS-169. Not sure why the link was not made...

Reply:
@gregaubert did point that PR out to me. It's solving a slightly different problem: if multiple reports are found, then PR #38 displays them all in a single report which looks like this:

![image](https://user-images.githubusercontent.com/10757559/56798312-70094f80-680e-11e9-8fc2-17131c456886.png)

The build summary above was produced by a build that analyses an MSBuild project, two Java projects, and has a single _Publish_ step.

Unfortunately, there's a problem if multiple MSBuild projects are analysed, which is what VSTS-169 addresses. The report file for MSBuild projects is always written to the same place on disk, so the only way to see a summary for each project is to have multiple _publish_ steps, which is presumably what the user who reported the issue is doing. 
With this change, the summary reports look like this:

![image](https://user-images.githubusercontent.com/10757559/56798766-5ddbe100-680f-11e9-99a7-9d5df8786cf0.png)

Without the project names, the summaries look identical.
Adding the project name doesn't change the vertical height of the widget. It might make it slightly wider, but there is nothing to the right of the widget that wouldn't doesn't matter.

This change is a simple fix to make the widgets look different. I've created a separate ticket to address the inconsistency in the handling of the reports: https://jira.sonarsource.com/browse/VSTS-191

Reply:
Make totally sense, thanks for the explanation.

Reply:
Apologies, I meant to call this out in a PR comment but forgot :-(